dashboard/modules/placement_group/placement_group_head.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+import aiohttp.web
+
+import ray.dashboard.utils as dashboard_utils
+import ray.dashboard.optional_utils as dashboard_optional_utils
+
+from ray.dashboard.optional_utils import rest_response
+
+logger = logging.getLogger(__name__)
+routes = dashboard_optional_utils.ClassMethodRouteTable
+
+
+class PlacementGroupHead(dashboard_utils.DashboardHeadModule):
+    def __init__(self, dashboard_head):
+        super().__init__(dashboard_head)
+
+    @routes.get("/api/v0/placement_groups")
```

Comment:
Originally, I was thinking to create a `state_head.py` file and add all routes in there, but it seems like the current codebase makes it difficult to do so. It is because it assumes each of module cannot communicate each other directly through the interface, and only data is shared. This means if we want to use some of data from the job module, it is not possible within the state module.

Reply:
I think the code will be structured in this way;

Routes (per each state)
------------------------
Post Processing (e.g., limit)
----------------------------
Aggregator (from agent, GCS, and raylet)

so grouping routes by aggregator seems a bit unnatural to me. Wdyt?

Reply:
No strong preference here: initially I was thinking it's nice to have all public apis in a single place so people can find them easily.